src/cascadia/TerminalApp/SettingsTab.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.

```

Comment:
Slap one of those doc comments with an abstract on here and put your name on it! :)

(this is the first one I noticed, but I'm almost at the end of my review. Could you check if you need to do the same with any of the other .h introduced too plz)